vcs/VTKPlots.py

```diff
@@ -626,10 +626,14 @@ def geometry(self, *args):
         y = args[1]
 
         if self.renWin is not None:
-            self.renWin.SetSize(x, y)
+            self.setsize(x, y)
         self._geometry = {'width': x, 'height': y}
         self._lastSize = (x, y)
 
+    def setsize(self, x, y):
```

Comment:
@danlipsa should we replace self.geometry with this? It seems both function aim at doing the same thing no?
see: https://github.com/UV-CDAT/vcs/blob/master/vcs/VTKPlots.py#L619

Reply:
geometry sets the _geometry and _lastSize as well - this is why I added a new function. Maybe when we get rid of bgX bgY we further clean up this.